pandas-stubs/core/groupby/groupby.pyi

```diff
@@ -139,9 +139,9 @@ class BaseGroupBy(SelectionMixin[NDFrameT], GroupByIndexingMixin):
     @final
     def __iter__(self) -> Iterator[tuple[Hashable, NDFrameT]]: ...
     @overload
-    def __getitem__(self: BaseGroupBy[DataFrame], key: Scalar | Hashable | tuple[Hashable, ...]) -> generic.SeriesGroupBy: ...  # type: ignore[overload-overlap]
```

Comment:
are there `Hashable` objects that do not conform to `Iterable[Hashable]` that pandas accepts? If I understand it correctly, they would now not be covered by pandas-stubs (that might be fine if Hashable was mostly meant as a catch call to avoid false positves)

Reply:
I removed `Hashable` here because it was overlapping with `slice`.  Now I have removed `slice`.  I could put `Hashable` back, but coming up with a test for that seems to be difficult.  Leaving it out doesn't break any of our existing tests.